client/my-sites/checkout/checkout/domain-refund-policy.jsx

```diff
@@ -73,11 +73,11 @@ class DomainRefundPolicy extends React.Component {
 			);
 		}
 
-		return message;
+		return { newDomainPolicy: message };
```

Comment:
This works... but it does seem to me that it makes the API of these functions harder to understand. Rather than returning a React node, which is very easy to use in any use-case, they now return an object with one key, which is not very obvious and of questionable usefulness.

What if we left these functions as they were, but put their results into an object with keys inside `getApplicablePolicies`?

Reply:
Addressed in 7ccbcb8